web-animations-1/Overview.bs

```diff
@@ -1150,7 +1125,11 @@ as CSS Animations [[CSS-ANIMATIONS-1]].
     condition from the following, if any:
 
     <div class="switch">
-
+    :   If |animation|'s <a>current time</a> is <a>unresolved</a>,
+        <a>start time</a> is <a>unresolved</a>, the associated <a>timeline</a>
+        is not [=monotonically increasing=] and the <var>auto-rewind</var> flag
```

Comment:
This needs to clarify what happens when there is no associated timeline.

Looking at this more, I feel a little unsure about this abstraction. If the animation has a negative playback rate, it seems like we should seek to the end of the animation. Maybe we should keep the conditions as-is, but alternative between setting the hold time or start time based on the type of timeline? Would that work?

Reply:
Changed per your suggestion.